deeplake/core/seed.py

```diff
@@ -11,7 +11,7 @@ def __new__(cls):
 
 
     def seed(self, seed: Optional[int] = None):
-        if isinstance(seed, Optional[int]):
+        if isinstance(seed, int):
```

Comment:
Seed here also can be None to reset previous values this this statement is not complete

Reply:
The fix for this will come through with the fix in main. I had to bring the branch into this PR early so the tests will pass

Reply:
main is merged in now with the final fix